README.md

```diff
@@ -39,50 +39,99 @@ new AShot()
   .takeScreenshot(webDriver);
 ```
 
-#####Capturing shots when a browser has fixed or variable header height
-When using AShot to capture screen of a device (iOS, Android) or it's simulator, then resulting image is returned with browser's header.
+#####Capturing the WebElement
 
-For example to capture page on iOS 7 device we will use fixed header's height of 98px.
+One can take a screenshot of the particular WebElement(s). Just specify the element(s).
+```java
+WebElement myWebElement = webDriver.findElement(By.cssSelector("#my_element"));
+new AShot()
+  .takeScreenshot(webDriver, myWebElement);
+```
+ 
+As noted earlier aShot will find element's size and position and crop original image. WebDriver API provides a method to find the WebElement's coordinates but different WebDriver implementations behave differently. In my opinion the most general approach to find coordinates is to use jQuery. So aShot uses jQuery by default. But some drivers have problems with Javascript execution such as OperaDriver. In this case there is another way to find the WebElement's coordinates.
+```java
+new AShot()
+  .coordsProvider(new WebDriverCoordsProvider()) //find coordinates with WebDriver API
+  .takeScreenshot(webDriver, myWebElement);
+```
+Feel free to implement own CoordsProvider. Pull requests are welcome.
 
+#####Capturing shots from browsers with pixel ratio > 1 (Retina displays)
+Easy to capture shots from HD screens when device pixel ratio is greater that one.
 ```java
-int browserHeaderHeight = 98;
 new AShot()
-  .shootingStrategy(new ViewportPastingStrategy(browserHeaderHeight))
+  .shootingStrategy(ShootingStrategies.scaling(2))
```

Comment:
It's better to move it to wiki cookbook. It is too complex information for readme